doc/eventhandler/counterhandler.rst

```diff
@@ -7,24 +7,39 @@ Counter Handler Module
 
 The counter event handler module is used to count certain events.
 You can define arbitrary counter names and each occurrence of an event will
-increase the counter in the counter table.
+modify the counter in the counter table according to the selected action.
 
 These counters can be used to graph time series of failed authentication, assigned tokens,
 user numbers or any other data with any condition over time.
 
 Possible Actions
 ~~~~~~~~~~~~~~~~
 
-increase_counter
-................
+.. describe:: increase_counter
```

Comment:
just thought it looks nicer :-)
Maybe we can change the sphinx config to avoid numbering all levels of subsections. It gets confusing if the length of the section number surpasses the text...

Reply:
You are right. Skipping chapter numbering for lower levels might make sense.
Just let us keep this in a consistent way through the documentation and not change this in this PR.